.travis.yml

```diff
@@ -40,16 +40,15 @@ before_install:
   - wget http://bit.ly/miniconda -O miniconda.sh
   - bash miniconda.sh -b -p $HOME/miniconda
   - export PATH="$HOME/miniconda/bin:$PATH"
-  - conda update conda --yes
+  - conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
```

Comment:
@lsetiawan this is unrelated but nice to have.

Reply:
What do these extra options do?

Reply:
- `always_yes yes` we can drop the `--yes` everywhere making the config cleaner
- `changeps1 no`  means that the env won't be shown in the prompt, which is good for CIs to avoid issues with the default prompt options there
- `show_channel_urls true` will display the channel origin for each package in the logs (good for debuging)